kie-drools-wb-parent/kie-drools-wb-home-page-community/src/main/java/org/kie/workbench/drools/client/home/HomeProducer.java

```diff
@@ -57,21 +76,34 @@ public HomeModel get() {
                                             LIBRARY));
         design.addLink(new HomeShortcutLink(translationService.format(Constants.Dashboards),
                                             BUSINESS_DASHBOARDS));
+        return design;
+    }
+
+    private HomeShortcut createDevOpsShortcut() {
+        final HomeShortcutLink deployments = new HomeShortcutLink(translationService.format(Constants.Deployments),
+                                                                  DEPLOYMENTS);
+        final HomeShortcutLink servers = new HomeShortcutLink(translationService.format(Constants.Servers),
+                                                              SERVER_MANAGEMENT);
+        final boolean isDeploymentsAuthorized = shortcutHelper.authorize(deployments.getPerspectiveIdentifier());
 
         final HomeShortcut devOps = ModelUtils.makeShortcut("fa fa-gears",
                                                             translationService.format(Constants.DevOps),
-                                                            translationService.format(Constants.DevOpsDescription),
+                                                            getDevOpsDescription(isDeploymentsAuthorized),
                                                             () -> placeManager.goTo(SERVER_MANAGEMENT),
                                                             SERVER_MANAGEMENT,
                                                             PERSPECTIVE);
-        devOps.addLink(new HomeShortcutLink(translationService.format(Constants.Deployments),
-                                            DEPLOYMENTS));
-        devOps.addLink(new HomeShortcutLink(translationService.format(Constants.Servers),
-                                            SERVER_MANAGEMENT));
+        if (isDeploymentsAuthorized) {
+            devOps.addLink(deployments);
+        }
+        devOps.addLink(servers);
 
-        model.addShortcut(design);
-        model.addShortcut(devOps);
+        return devOps;
+    }
 
-        return model;
+    private String getDevOpsDescription(final boolean isDeploymentsAuthorized) {
```

Comment:
I see this method 4 times in the PR, any chance to get it into some util class? Sure, I have not investigated deeper if it is possible. So if not applicable, I am fine.